pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -1991,6 +2038,35 @@
       ^-  ?
       &(=(item-binding binding) =(item-duct duct))
     ==
+  ::  +add-cache: remember a cache mapping
+  ::
+  ++  add-cache
+    |=  [=path =cache-entry]
+    ^-  [(list move) server-state]
+    =.  cache.state  (~(put of cache.state) path cache-entry)
+    `state
+  ::  +del-cache: forget a cache mapping
+  ::
+  ++  del-cache
+    |=  =path
+    ^-  [(list move) server-state]
+    =.  cache.state  (~(del of cache.state) path)
+    `state
+  ::  +add-cache-binding: remember a cache binding
+  ::
+  ++  add-cache-binding
+    |=  [url=@t =path]
+    ^-  [(list move) server-state]
+    =.  cache-bindings.state  (~(put by cache-bindings.state) url path)
```

Comment:
the cache-bindings are not meant to be referentially transparent, so this is good as is